pom.xml

```diff
@@ -112,16 +112,18 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <!--
+            FIX ME: Have to comment out this plugin since it conflicts with ForkJoinPool dependency.
```

Comment:
Do we really? Maybe up the requirement to Java 7? If not, unfortunately we may have to load plugins sequentially in SoapUI OS.